[DOC-588][docs] Voyager docs for v1.8.8 (#25415)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

The `export data from target` command may result in duplicated events if you restart Voyager, or there is a change in the YugabyteDB database server state. Consequently, the [get data-migration-report](#get-data-migration-report) command may display additional events that have been exported from the target YugabyteDB database, and imported into the source-replica or source database. For such situations, it is recommended to manually verify data in the target and source-replica, or source database to ensure accuracy and consistency.
       {{</note>}}

> 1. If there are any `NOT VALID` constraints on the source, create them after the `import data` command is completed by using the `import schema` command with the `post-snapshot-import` flag:

Author: ```suggestion
1. If the source has any NOT VALID constraints, after the `import data` command has completed, create them by running `import schema` with the `post-snapshot-import` flag:
```
